lib/ansible/modules/source_control/git.py

```diff
@@ -223,6 +232,13 @@
     repo: https://github.com/ansible/ansible-examples.git
     dest: /src/ansible-examples
     separate_git_dir: /src/ansible-examples.git
+
+# Example clone a repo with --depth and --no-single-branch option
+- git:
+    repo: https://github.com/ansible/ansible-examples.git
+    dest: /srv/checkout
+    depth: 1
+    sinble_branch: no
```

Comment:
single is spelled wrong.

Reply:
Thank you for checking. I have fixed it :)